pkgs/tools/misc/pdf-parser/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, python3Packages, fetchzip }:
+
+python3Packages.buildPythonApplication {
+  pname = "pdf-parser";
+  version = "0.7.4";
+
+  src = fetchzip {
+    url = https://didierstevens.com/files/software/pdf-parser_V0_7_4.zip;
+    sha256 = "1j39yww2yl4cav8xgd4zfl5jchbbkvffnrynkamkzvz9dd5np2mh";
+  };
+
+  format = "other";
+
+  installPhase = ''
+    install -Dm555 pdf-parser.py $out/bin/pdf-parser.py
+  '';
+
+  preFixup = ''
+    substituteInPlace $out/bin/pdf-parser.py \
+      --replace '/usr/bin/python' '${python3Packages.python}/bin/python'
```

Comment:
Is this necessary? I would expect `buildPythonApplication` to fix this automatically.

Reply:
The `buildPythonApplication` only fixes the shebang at the first line of the script.
But without this line, there is still a `/usr/bin/python` in the script.

Reply:
IMHO `buildPythonApplication` shouldn't touch the code itself and it's perfectly fine to manually fix those paths in the sources, so LGTM.